pyproject.toml

```diff
@@ -6,7 +6,7 @@ authors = ["@mozilla/jbi-core"]
 license = "MPL"
 
 [tool.poetry.dependencies]
-python = ">=3.10 <3.11"
```

Comment:
Instead of a range, should we pin this at a major and minor version? We won't expect this to be run on a variety of Python versions since we're only using whatever the Python version of the container is

Reply:
Agree. The only advantage is for developers when running things locally.